core/docs/API_Python.md

```diff
@@ -168,6 +168,26 @@ Parameters
 Simulation
 ----------
 
+The available `method_type`s are:
+
+| Method                        | Argument |
+| ----------------------------- | :------: |
+| Landau-Lifshitz-Gilbert       | `"LLG"`  |
+| Geodesic Nudged Elastic Band  | `"GNEB"` |
+| Monte-Carlo                   | `"MC"`   |
+| Minimum Mode Following        | `"MMF"`  |
+
+The available `optimizer_type`s are:
+
+| optimizer                     | Argument |
+| ----------------------------- | :------: |
+| Semi-Implicit Method          | `"SIB"`  |
+| Semi-Implicit Method Modified | `"SIB2"` |
```

Comment:
The SIB2 Method is not meant to be used and will be removed at some point. It should not be mentioned here.